File: Leet-code/easy/1480. Running Sum of 1d Array.php
class Solution {

/**
 * @param Integer[] $nums
 * @return Integer[]
 */
function runningSum($nums) {


    $sum = 0;
    $length = count($nums);
    $data = [];

    for($i = 0; $i < $length; $i++){

            $sum += $nums[$i];

            $data[] = $sum;
        }

   return $data;
}
}

File: Leet-code/1480-running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
     
        int sum = 0;
        vector <int>data(nums.size());
        
            for(int i= 0; i<nums.size(); i++){
                sum += nums[i];
                data[i]= sum;
            }
        return data;
    }
};